wagtail/admin/ui/menus/__init__.py

```diff
@@ -0,0 +1,25 @@
+class MenuItem:
```

Comment:
I'm not quite sure if this should live here. The majority of `wagtail.admin.ui` is for renderable components, with the exception of `wagtail.admin.ui.sidebar`. Not to mention we already have `MenuItem` in that module too...

Maybe it's the `Button` classes that need to be here instead. Having them in a module called `widgets` isn't quite right, as it might be confused with the concept of form widgets. But that's where the classes have lived for many years, so I'm hesitant if we need to change that now.